extension/scripts/global/functions/containers.js

```diff
@@ -42,7 +43,7 @@ function createContainer(title, options = {}) {
 		containerClasses.push(theme.containerClass);
 		const container = document.newElement({ type: "div", class: containerClasses.join(" "), id: options.id });
 
-		const collapsed = options.collapsible && (options.id in filters.containers ? filters.containers[options.id] : false);
+		const collapsed = options.collapsed || (options.collapsible && (options.id in filters.containers ? filters.containers[options.id] : false));

```

Comment:
`collapsible` wasn't enough?

Reply:
Oh, that took me a while to understand. Not sure that 'collapsed' is the best name for that though.

Reply:
Added (and used) `onlyHeader`.